package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0",
+  "private": true,
+  "scripts": {
+    "build:css": "sass ./app/assets/stylesheets/tranzito_utils.scss ./app/assets/builds/tranzito_utils-compiled.css --no-source-map --load-path=node_modules"
```

Comment:
Is there a good reason not to put this in the `tranzito_utils_js` package.json?

Reply:
The `tranzito_utils_js` is an npm package and the styles belonged to the `tranzito_utils` gem that's why I added it here.

Reply:
But the npm package overlaps a lot with the gem! I think I will be confused by the multiple `package.json`s in the future.

If the scss build is included in the `tranzito_utils_js` npm package, will it make the bundled js much larger?

Reply:
I think two package.json make sense for the use-case when we need to add the JS files for the gem as well. Then we need to compile those JS files separately and might need separately compiled files.
I think for now It would be fine if we use only `tranzito_utils_js`.